packages/commonwealth/client/scripts/views/components/sidebar/CommunitySection/AddressItem.tsx

```diff
@@ -0,0 +1,84 @@
+import { WalletId } from '@hicommonwealth/shared';
+import { formatAddressShort } from 'client/scripts/helpers';
+import AddressInfo from 'client/scripts/models/AddressInfo';
+import NewProfile from 'client/scripts/models/NewProfile';
+import {
+  handleMouseEnter,
+  handleMouseLeave,
+} from 'client/scripts/views/menus/utils';
+import useAuthentication from 'client/scripts/views/modals/AuthModal/useAuthentication';
+import React from 'react';
+import { PopoverMenu } from '../../component_kit/CWPopoverMenu';
+import { CWIconButton } from '../../component_kit/cw_icon_button';
+import { CWIcon } from '../../component_kit/cw_icons/cw_icon';
+import { CWText } from '../../component_kit/cw_text';
+import { CWTooltip } from '../../component_kit/new_designs/CWTooltip';
+
+import './AddressItem.scss';
+
+type AddressItemProps = {
+  profile: NewProfile;
+  addressInfo: AddressInfo;
+  toggleRemoveModal: (val: boolean, address: AddressInfo) => void;
+  isSelected: boolean;
+};
+
+const AddressItem = (props: AddressItemProps) => {
+  const { addressInfo, toggleRemoveModal, isSelected } = props;
+  const { address, walletId } = addressInfo;
+
+  const { openMagicWallet } = useAuthentication({});
+
+  return (
+    <div className="AddressItem">
```

Comment:
you should use existing components like CWIdentificationTag or CWTag.

Take a look on TagsShowcase component for a clue.

https://common.xyz/components

![image](https://github.com/user-attachments/assets/391c47e4-7aa0-4f94-8410-3ecdd68cc0ad)